app/controllers/courses_controller.rb

```diff
@@ -1,10 +1,9 @@
 class CoursesController < ApplicationController
   before_action :set_course, except: %i(new create show)
-  before_action :authenticate_user!
   before_action :check_permitted_or_not , only: %i(edit destroy)
 
   def index
-    @courses = Course.recent_courses.page params[:page]
+    @course = Course.recent_courses.page params[:page]
```

Comment:
```suggestion
    @courses = Course.recent_courses.page params[:page]
```
should s because has many records